platform-core-module/src/main/java/com/softicar/platform/core/module/test/fixture/CoreModuleTestFixtureMethods.java

```diff
@@ -75,24 +75,26 @@ default AGUserPassword insertPassword(AGUser user, String password) {
 
 	// ------------------------------ localization ------------------------------ //
 
-	default AGLocalization insertLocalizationPreset(String name, LanguageEnum language, String decimalSeparator, String digitGroupSeparator) {
+	default AGLocalization insertLocalizationPreset(String name, LanguageEnum language, String decimalSeparator, String digitGroupSeparator,
+			String dateFormat) {
 
 		return new AGLocalization()//
 			.setName(name)
 			.setLanguage(AGCoreLanguage.getByLanguageEnum(language).get())
 			.setDecimalSeparator(decimalSeparator)
 			.setDigitGroupSeparator(digitGroupSeparator)
+			.setDateFormat(dateFormat)
 			.save();
 	}
 
 	default AGLocalization insertLocalizationPresetGermany() {
 
-		return insertLocalizationPreset("Deutschland", LanguageEnum.GERMAN, ",", ".");
+		return insertLocalizationPreset("Deutschland", LanguageEnum.GERMAN, ",", ".", "dd.MM.yyyy");
 	}
 
 	default AGLocalization insertLocalizationPresetUsa() {
```

Comment:
- actually `yyyy-MM-dd` is not USA, that would be rather `MM/dd/yyyy`
- we might also rename this method to `insertLocalizationPresetInterantional` and the locale `International`

Reply:
changed to `International`... I think nobody should ever use that nonsensical `MM/dd/yyyy` format

Reply:
LOL, so are imperial units like square feet, still they are used :D